spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  it 'add some comments' do
+    author = User.new(name: 'Tom', photo: 'Tom.png', bio: 'bio', posts_counter: 0)
+    post = Post.new(title: 'New post', text: 'Good evening', author:, like_counter: 0, comments_counter: 0)
+    post.save!
+
+    comment_creator = User.new(name: 'Jerry', photo: 'Tom.png', bio: 'bio', posts_counter: 0)
+    post.comments.create!(text: 'Hello World', author: comment_creator)
+    post.comments.create!(text: 'This is my second post', author: comment_creator)
+    expect(post.comments.length).to eql(2)
+  end
+end
```

Comment:
- [ ]  Kindly add a test for the `update_comments_counter` method you have in your Comment model as it is required to test all the custom methods in each model.